src/lib/Scenes/Artwork/Components/ImageCarousel/FullScreen/VerticalSwipeToDismiss.tsx

```diff
@@ -87,7 +87,7 @@ export const VerticalSwipeToDismiss: React.FC<{ onClose(): void }> = observer(({
       style={[{ opacity }]}
       snapToInterval={screenHeight}
       decelerationRate="fast"
-      scrollEventThrottle={16}
+      scrollEventThrottle={0.000000001}
```

Comment:
Why is this needed?

Reply:
unfortunately this throttles both native-side events and over-the-bridge events. We need to get updates for every single scroll event on the native side. I'd seen '16' mentioned in the docs as being the value to pass if you want to opt in to all scroll events, but after debugging I found out that it still drops quite a few events, which is bad news if you're animating things in unison with a scroll view. Setting the throttle to '0' has weird special semantics (only triggers one event at the start of a scroll), so the next best thing is a number very very close to 0.

One of the PRs I want to make to RN after this work is to make it so that this throttle doesn't necessarily apply to both events coming over the bridge and events on the native-side. Ain't nobody wants to throttle events with useNativeDriver 🤷‍♀

Reply:
I see. Perhaps it could be a `SEND_ALL_EVENTS` constant?

Reply:
maybe, let's see!